make/autoconf/jdk-version.m4

```diff
@@ -115,58 +119,63 @@ AC_DEFUN_ONCE([JDKVER_SETUP_JDK_VERSION_NUMBERS],
   AC_SUBST(COMPANY_NAME)
 
   # The vendor URL, if any
-  AC_ARG_WITH(vendor-url, [AS_HELP_STRING([--with-vendor-url],
-      [Set the 'java.vendor.url' system property @<:@not specified@:>@])])
-  if test "x$with_vendor_url" = xyes || test "x$with_vendor_url" = xno; then
-    AC_MSG_ERROR([--with-vendor-url must have a value])
-  elif [ ! [[ $with_vendor_url =~ ^[[:print:]]*$ ]] ]; then
-    AC_MSG_ERROR([--with-vendor-url contains non-printing characters: $with_vendor_url])
-  elif test "x$with_vendor_url" != x; then
-    # Only set VENDOR_URL if '--with-vendor-url' was used and is not empty.
-    # Otherwise we will use the value from "branding.conf" included above.
-    VENDOR_URL="$with_vendor_url"
-  fi
+  # Only set VENDOR_URL if '--with-vendor-url' was used and is not empty.
+  # Otherwise we will use the value from "branding.conf" included above.
```

Comment:
I think this is actually a bit confusing, especially when put in focus like this by your rewrite. Perhaps the values in branding.conf should be renamed like `DEFAULT_VENDOR_URL`? That is what it is, after all -- even if you modify branding.conf in a fork, it can still be overridden by configure arguments.

(This comment is also driven by the fact that I don't really like a construction where the same value both goes into the UTIL_ARG_WITH as comes out of it. If at all possible, I really like to avoid re-assigning values to variables).

Reply:
Alright, I avoided changing those to keep the commit as small as possible, but I guess it's free game now that I have the green light